dubbo-registry/dubbo-registry-api/src/main/java/org/apache/dubbo/registry/client/event/listener/ServiceInstancesChangedListener.java

```diff
@@ -473,12 +467,12 @@ public boolean equals(Object o) {
             return false;
         }
         ServiceInstancesChangedListener that = (ServiceInstancesChangedListener) o;
-        return Objects.equals(getServiceNames(), that.getServiceNames()) && Objects.equals(listeners, that.listeners);
+        return Objects.equals(getServiceNames(), that.getServiceNames());
```

Comment:
In different ApplicationModel, listeners are different

Reply:
@AlbumenJ Should the listeners parameter be retained in equals, or should we use the ApplicationModel to determine

Reply:
> listeners parameter be retained in equals

ok